tests/metrics/test_compare_ground_truth.py

```diff
@@ -0,0 +1,289 @@
+    generate_pred(start_index=500, end_index=520, label="State", text="2"),
+    generate_pred(start_index=600, end_index=620, label="State", text="3"),
+    generate_pred(start_index=100, end_index=106, label="Zip", text="1"),
+    generate_pred(start_index=200, end_index=206, label="Zip", text="2"),
+    generate_pred(start_index=300, end_index=306, label="Zip", text="3"),
+    generate_pred(start_index=400, end_index=406, label="Zip", text="4"),
+    generate_pred(start_index=500, end_index=506, label="Zip", text="5"),
+    generate_pred(start_index=600, end_index=606, label="Zip", text="6"),
+]
+predictions_list = [
+    generate_pred(start_index=3, end_index=7, label="Vendor Name", text="A"),
+    generate_pred(start_index=32, end_index=41, label="Vendor Name", text="B"),
+    generate_pred(start_index=13, end_index=17, label="Amount", text="A"),
+    generate_pred(start_index=18, end_index=22, label="Amount", text="B"),
+    generate_pred(start_index=42, end_index=51, label="Amount", text="C"),
+    generate_pred(start_index=13, end_index=17, label="Date", text="A"),
+    generate_pred(start_index=18, end_index=22, label="Date", text="B"),
+    generate_pred(start_index=42, end_index=51, label="Date", text="C"),
+    generate_pred(start_index=10, end_index=35, label="Address", text="A"),
+    generate_pred(start_index=500, end_index=522, label="Address", text="B"),
+    generate_pred(start_index=600, end_index=618, label="Address", text="C"),
+    generate_pred(start_index=36, end_index=40, label="Business Category", text="A"),
+    generate_pred(start_index=61, end_index=70, label="Business Category", text="B"),
+    generate_pred(start_index=80, end_index=90, label="Business Category", text="C"),
+    generate_pred(start_index=700, end_index=710, label="Business Category", text="D"),
+    generate_pred(start_index=36, end_index=40, label="PO Number", text="A"),
+    generate_pred(start_index=61, end_index=70, label="PO Number", text="B"),
+    generate_pred(start_index=80, end_index=90, label="PO Number", text="C"),
+    generate_pred(start_index=700, end_index=710, label="PO Number", text="D"),
+    generate_pred(start_index=10, end_index=35, label="State", text="A"),
+    generate_pred(start_index=500, end_index=522, label="State", text="B"),
+    generate_pred(start_index=100, end_index=106, label="Zip", text="A"),
+    generate_pred(start_index=200, end_index=206, label="Zip", text="B"),
+    generate_pred(start_index=300, end_index=306, label="Zip", text="C"),
+    generate_pred(start_index=400, end_index=406, label="Zip", text="D"),
+    generate_pred(start_index=500, end_index=506, label="Zip", text="E"),
+    generate_pred(start_index=600, end_index=606, label="Zip", text="F"),
+    generate_pred(start_index=700, end_index=706, label="Zip", text="G"),
+    generate_pred(start_index=800, end_index=806, label="Zip", text="H"),
+    generate_pred(start_index=900, end_index=906, label="Zip", text="H"),
+]
+expected_labels = [
```

Comment:
should be able to get this based on your input within the test, don't need outside of test (but if you did would want it in all caps EXPECTED_LABELS